README.Rmd

```diff
@@ -8,18 +8,120 @@ output: github_document
 knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>",
-  fig.path = "README/README-"
+  fig.path = "README-"
 )
 ```
 
 # googledrive
 
-[![Build Status](https://travis-ci.org/tidyverse/googledrive.svg?branch=master)](https://travis-ci.org/tidyverse/googledrive)[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/LucyMcGowan/googledrive?branch=master&svg=true)](https://ci.appveyor.com/project/LucyMcGowan/googledrive)[![Coverage Status](https://img.shields.io/codecov/c/github/tidyverse/googledrive/master.svg)](https://codecov.io/github/tidyverse/googledrive?branch=master)
+[![Build Status](https://travis-ci.org/tidyverse/googledrive.svg?branch=master)](https://travis-ci.org/tidyverse/googledrive)[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/tidyverse/googledrive?branch=master&svg=true)](https://ci.appveyor.com/project/tidyverse/googledrive)[![Coverage Status](https://img.shields.io/codecov/c/github/tidyverse/googledrive/master.svg)](https://codecov.io/github/tidyverse/googledrive?branch=master)
 
 `r emo::ji("construction")` WARNING: this is very much under construction `r emo::ji("construction")`
 
-## installation
+## Overview
+
+`googledrive` interfaces with Google Drive from R, allowing users to seamlessly manage files on Google Drive from the comfort of their console `r emo::ji("house")`. 
+
```

Comment:
Technically, you'll need to suggest emo if you do this and it may not be worth it?